tasks/install.json

```diff
@@ -48,6 +48,7 @@
     },
     "password": {
       "description": "The password to use when downloading from a source location requiring authentication",
+      "sensitive": true,
```

Comment:
Thanks @jordanbreen28 could you do the same for the windows install_powershell.json?

Reply:
hey josh, so [install_powershell.json](https://github.com/puppetlabs/puppetlabs-puppet_agent/blob/main/tasks/install_powershell.json#L52) & [install_shell.json](https://github.com/puppetlabs/puppetlabs-puppet_agent/blob/main/tasks/install_shell.json#L53) already have password defined as `Optional[Sensitive[String[1]]]`, so wasn't sure if that was necessary. Is that the case? (Forgive me, I am not that versed when it comes to bolt..) 

[install.json](https://github.com/puppetlabs/puppetlabs-puppet_agent/blob/main/tasks/install.json#L51) seemed to be the only place where password was not marked as sensitive using the meta param, or datatype.

Reply:
hey @joshcooper - I updated all three instances of password. Thanks for pointing that out!

Reply:
hey @joshcooper - I updated all three instances of the password param to be sensitive. Thanks for pointing that out!